rules/integrations/aws/exfiltration_ec2_ami_shared_with_separate_account.toml

```diff
@@ -0,0 +1,101 @@
+[metadata]
+creation_date = "2024/04/16"
+integration = ["aws"]
+maturity = "production"
+min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
+min_stack_version = "8.9.0"
+updated_date = "2024/05/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an AWS Amazon Machine Image (AMI) being shared with another AWS account. Adversaries with access may share an
+AMI with an external AWS account as a means of data exfiltration. AMIs can contain secrets, bash histories, code
+artifacts, and other sensitive data that adversaries may abuse if shared with unauthorized accounts. AMIs can be made
+publicly available accidentally as well.
+"""
+false_positives = [
+    """
+    AMI sharing is a common practice in AWS environments. Ensure that the sharing is authorized before taking action.
+    """,
+]
+from = "now-60m"
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS EC2 AMI Shared with Another Account"
```

Comment:
```suggestion
name = "EC2 AMI Shared with Another Account"
```

Reply:
Isn't EC2 always Amazon? Could potentially shorten all file names by removing AWS, as it is indirectly implied to be AWS. Up to you!

Reply:
Yes, it's a very good point. The only reason I keep them is from a free form searching perspective in the UI. Ideally, the searching capability is robust enough. I'll remove from here on out.